formats/cbor/commonMain/src/kotlinx/serialization/cbor/Cbor.kt

```diff
@@ -212,6 +226,7 @@ public class Cbor(
         override fun decodeBoolean() = decoder.nextBoolean()
 
         override fun decodeByte() = decoder.nextNumber().toByte()
+        override fun decodeByteString(): ByteString = ByteString(decoder.nextByteString())
```

Comment:
nitpick: Return type is redundant here. More so, we should follow convention of the functions near it.

```suggestion
        override fun decodeByteString() = ByteString(decoder.nextByteString())
```